engineyard/ey-cookbooks-dev-v7
# [FBZ-10423] Migrate DB Admin Tools Recipe to v7

**Description of your patch**

Adds DB Admin Tools by migrating recipe from v6 to v7

**Estimated risk**

High risk. Failure will compromise DB slave setup

**Components involved**

`ey-app/metadata.rb`
`ey-db_master/metadata.rb`
`ey-db_replica/metadata.rb`
`ey-app/recipes/build.rb`
`ey-db_master/recipes/prep.rb`

**Dependencies**

`ey-db_master`
`ey-db_replica`

**Description of testing done**
* Booted up fresh Environment with Database Master and Database Slave (Tested with PostgreSQL)
* Uploaded recipe with ey-db_admin_tools included
* Triggered chef run
* Verified that the scripts are configured and scheduled to run 

**QA Instructions**

* Create environment with Database Master and Database Slave
* Uploaded recipe with ey-db_admin_tools included
* Apply changes on the environment
* Verify that the scripts are configured and scheduled to run
